### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by ridhachahed** [nauyan/Segmentation#12]
Hello,

Did you preprocess the data because I get different results with your data than the official release. Also it's not in the same format (.png vs .tif)

**Comment by nauyan** [comment]
@ridhachahed yes after pushing the code I did realize that I didn't use the official format of the Monuseg. The dataset I have used can be found here: https://github.com/nauyan/Segmentation/blob/master/Datasets/MonuSeg-20200319T073151Z-001.zip

**Comment by ridhachahed** [comment]
I am curious to know the difference between the two because I get better results on yours. Did you apply any transformations to it?

**Comment by nauyan** [comment]
No I haven't applied any transformations on this one.

**Comment by ridhachahed** [comment]
I found out that images have the same names in the two datasets but are actually different and have different annotations :)

**Comment by tbnb1** [comment]
I would like to ask why the original data set I downloaded from the official website, the training set has 37 data, while your training set is 30 data
